core/src/main/java/org/apache/accumulo/core/client/impl/ScannerOptions.java

```diff
@@ -244,4 +247,20 @@ public void setBatchTimeout(long timeout, TimeUnit timeUnit) {
   public long getBatchTimeout(TimeUnit timeUnit) {
     return timeUnit.convert(batchTimeOut, TimeUnit.MILLISECONDS);
   }
+
+  @Override
+  public void setContext(String context) {
+    this.context = context;
```

Comment:
I could, but it's not an error to pass null. The reference is initialized to null and is used successfully that way until it's set to something not null. Having said that, I don't really care one way or the other, just curious if we are forcing the user to use the api in a particular way.